shesmu-server/src/main/java/ca/on/oicr/gsi/shesmu/server/BaseSimulateRequest.java

```diff
@@ -73,6 +74,13 @@
 import org.objectweb.asm.commons.Method;
 
 public abstract class BaseSimulateRequest {
+  private static final Counter CACHE_REFRESH =
+      Counter.build(
+              "shesmu_simulate_refresh",
+              "The number of simulation requests and whether the cache refresh should be"
```

Comment:
'should be triggered' suggests an action needed on the part of anyone looking at the metrics imo. 'and the cache freshness requested'? i don't know if there's particular tense conventions for metric descriptions

Reply:
Okay. I am bad at writing these.